spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe PasswordResetsController do
+  describe "POST #create" do
+    let(:user)      { create(:confirmed_user) }
+    let(:params)    { { password_reset: { email: user.email } } }
+    let(:arguments) { { email: user.email } }
+    let(:context)   { double(:context, success?: true) }
+
+    before(:example) do
+      allow(PasswordResetOrganizer).to receive(:call).with(arguments).
+        and_return(context)
+    end
+
+    it "calls PasswordResetOrganizer" do
+      expect(PasswordResetOrganizer).to receive(:call)
```

Comment:
Haha, I usually break things before getting it right too! :smile: 

Think about the linguistic meaning in English. We are expecting something **to receive**. I interpret it as, the "something" hasn't received "it" yet but we expect that it will in the future. :stuck_out_tongue: 

RSpec offers the counterpart matcher though, `expect(...).to have_received(...)`. In that case you would do whatever it is that should send the method call to the object **before** the expectation.